src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -52,7 +52,7 @@
         "@autorest/csharp": "3.0.0-beta.20230309.1",
         "@azure-tools/typespec-autorest": "0.29.0",
         "@azure-tools/typespec-azure-core": "0.29.0",
-        "@azure-tools/typespec-client-generator-core": "0.29.0",
+        "@azure-tools/typespec-client-generator-core": "0.30.0-dev.13",
```

Comment:
Shall we wait for the official build of TCGC?

Reply:
Then we don't have tests for this feature. We could change to official one after it releases.